source/as-stream.js

```diff
@@ -10,6 +10,8 @@ module.exports = options => {
 	const output = new PassThrough();
 	const proxy = duplexer3(input, output);
 
+	options.gotRetries.retry = () => 0;
```

Comment:
Why is this not just `options.gotRetries = 0;`?

Reply:
The tests would fail then, because `normalizeArguments` gets called before and makes a function. Calling `options.gotRetries.retry(++retryCount, error)` would throw because `options.gotRetries.retry` is not a `function`. So it must be done like that.

Reply:
👍